### ISSUE TITLE
Option for JMM Server to remove files from MyList when deleted from HDD.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dizzydezz/jmm#138]
```
To give an example, I used to have NPC's 720p release of The Tatami Galaxy. 
This was added to MyList as "collecting." After I watched several episodes, I 
deleted NPC's 720p from my hard drive and replaced it with the 1080p release. I 
expect JMM Server to automatically remove the 720p files from MyList, or at 
least set their file state to "deleted." Right now, it does neither, and simply 
lets duplicates sit in MyList, which screws up stats as well as updating via 
IRC.

```

Original issue reported on code.google.com by `aznzh...@gmail.com` on 27 Jan 2012 at 7:18

**Comment by GoogleCodeExporter** [comment]
```
You can already do this on the Utilities tab.
"Missing MyList files"

Also if you delete the files using JMM Desktop, they will also be removed from 
your list
```

Original comment by `werndly...@gmail.com` on 28 Jan 2012 at 12:11

**Comment by GoogleCodeExporter** [comment]
```
Oh, sorry. I'm an idiot.

But... this doesn't change the state to deleted, it actually removes the file? 
Is there a way I can specify the first option?
```

Original comment by `aznzh...@gmail.com` on 28 Jan 2012 at 12:46

**Comment by GoogleCodeExporter** [comment]
```
I will make it an option to mark as deleted instead of removing the file from 
the list (as long as the API supports it)
```

Original comment by `werndly...@gmail.com` on 29 Jan 2012 at 12:00
- Changed state: **Accepted**
- Added labels: **Milestone-Release0.9_Beta**, **Product-JMM**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.1.26

The option is available under JMM Desktop > Settings > AniDB > Delete Action
```

Original comment by `werndly...@gmail.com` on 13 Dec 2012 at 10:59
- Changed state: **Fixed**
